### PR TITLE
Update nf-develop

### PR DESCRIPTION
**Pull request by christopher-hakkaart**
- Updated the environment so TUI works
- Updated exercises to use new TUI
- Changed the naming of the example pipeline
- Added more commands to move between directories
- Removed GitHub exercises
- Fix misc typos and dead links

### CODE REVIEW

File: docs/nf_develop/1_04_parameters.md [hunk 1]
```diff
@@ -115,23 +123,32 @@ The parameters that you have added to your pipeline will be added to the bottom
...
+!!!note
+
+    Ungrouped parameters in schema will cause a warning.
+
 Once you have made your edits you can click `Finished` and all changes will be automatically added to your `nextflow_schema.json` file.
 
 !!! question "Exercise"
 
-    Use the `nf-core schema build` command to update your schema. Add any grouping and information you think is appropriate.
+    Execute the `nf-core pipelines schema build` commanddd to update your schema.
```
Comment (maxulysse) [inline docs/nf_develop/1_04_parameters.md:134]:
```suggestion
    Execute the `nf-core pipelines schema build` command to update your schema.
```

Comment (christopher-hakkaart) [reply docs/nf_develop/1_04_parameters.md:134]: Thanks!

**Review by kenibrewer — COMMENTED** [review]
One perfectly innocent typo that needs correcting. Otherwise LGTM.

File: docs/nf_develop/1_03_pipeline.md [hunk 2]
```diff
@@ -108,26 +121,30 @@ INFO     Installing 'seqtk/trim'
 INFO     Use the following statement to include this module:
 
 include { SEQTK_TRIM } from '../modules/nf-core/seqtk/trim/main'
-
 ```
 
 !!! question "Exercise"
 
-    Run the `nf-core modules install` command to add the `seqtk/trim` module to your pipeline.
+    Run the `nf-core modules install` ass the `seqtk/trim` module to your pipeline.
```
[the remark this answers is not in the document]
Comment (christopher-hakkaart) [reply docs/nf_develop/1_03_pipeline.md:128]: I knew what I was saying 😆

**Review by kenibrewer — APPROVED** [review]

**Comment by christopher-hakkaart** [comment]
Thanks @kenibrewer

**Pull request merged**